.travis.yml

```diff
@@ -5,7 +5,7 @@ language: scala
 scala:
   - 2.10.7
   - 2.11.12
-  - 2.12.6
+  - 2.12.8
```

Comment:
Should there be a 2.13 build here, or are we waiting for the final release?

Reply:
Probably. I don't do the real signed builds through Travis, but it would make sense to have it here.